docs/source/auth_rules.md

```diff
@@ -285,6 +285,24 @@
     <td><sub>No one can edit existing Schema</sub></td>
     <td><sub>Editing a Schema</sub></td>
   </tr>
+  <tr>
+    <td><sub>SET_CONTEXT</sub></td>
+    <td><sub>ADD</sub></td>
+    <td><sub><code>*</code></sub></td>
+    <td><sub><code>*</code></sub></td>
+    <td><sub><code>*</code></sub></td>
+    <td><sub>1 TRUSTEE OR 1 STEWARD OR 1 ENDORSER</sub></td>
+    <td><sub>Adding a new Schema</sub></td>
```

Comment:
I believe it needs to be `Adding a new Context`

Reply:
fixed in [ff6f6d7](https://github.com/hyperledger/indy-node/pull/1430/commits/ff6f6d7db1275f4eaadf43cf17127f8c82390936)